Cargo.toml

```diff
@@ -29,6 +29,7 @@ error-chain = { version = "0.12.4", default-features = false }
 reqwest = { version = "0.11.4", features = ["blocking", "json"] }
 tungstenite = { version = "0.17.3", features = ["native-tls"] }
 url = "2.2.2"
+tokio = { version = "1.21.2", features = ["full"] }
```

Comment:
yes, it's required because it's most popular (and efficient?) way to create async app in rust. 

also I updated my [PR](https://github.com/wisespace-io/binance-rs/pull/154) that was the inspiration of this PR

Reply:
i can't see anywhere in the library that `tokio` is used directly. It is only used in the examples and tests. If this is correct, then it's only a `dev` dependency, not a regular dependency.
I understand that it's a transitive dependency (of `reqwest`, probably among others), but since it's not a direct dependency you're over-constraining the dependency resolution by including it